integration/spark/src/main/scala/org/apache/spark/sql/CarbonSession.scala

```diff
@@ -40,12 +40,18 @@ import org.apache.carbondata.streaming.CarbonStreamingQueryListener
  * Session implementation for {org.apache.spark.sql.SparkSession}
  * Implemented this class only to use our own SQL DDL commands.
  * User needs to use {CarbonSession.getOrCreateCarbon} to create Carbon session.
+ *
+ * @deprecated Since 2.0, only use for backward compatibility,
+ *             please switch to use {@link CarbonExtensions}.
  */
+@Deprecated
 class CarbonSession(@transient val sc: SparkContext,
```

Comment:
"**stored by**" syntax also we need to deprecate and update document for the same?

Reply:
the document already contains it

Reply:
please check https://github.com/apache/carbondata/blob/master/docs/quick-start-guide.md,
it lists all supported syntax by CarbonSession and CarbonExtensions.